packages/terra-application-navigation/CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ## Unreleased
 
+* Changed
```

Comment:
```suggestion
* Added
```
Since we are adding new props and not making changes to existing props.

Reply:
Switched wording to "added"